### ISSUE TITLE
Usage of co_attention_mask

### ISSUE THREAD

**Issue by tejas1995** [facebookresearch/vilbert-multi-task#44]
I'm trying to extract representations from pre-trained VILBert, by building a method within `class VILBertForVLTasks` that returns `sequence_output_t, sequence_output_v, pooled_output_t, pooled_output_v` (line 1652 of vilbert/vilbert.py). I want the text representations (`sequence_output_t`) to be independent of the visual input, which I figured I would need to do using the `co_attention_mask` input. 

The default `co_attention_mask` is None, do I need to set it a tensor of ones of the appropriate size in order to mask the text and visual inputs from each other? I tried following the usage of co_attention_mask in BertModel, BertEncoder etc, but it's not clear exactly where this masking is applied, or where the variable `use_co_attention_mask` is set to True.
